doc/source/templates/configs/compute/gpu/aws.yaml

```diff
@@ -1,11 +1,11 @@
-# 4 g4dn.2xlarge nodes --> 32 CPUs, 4 GPUs
+# 3 g5.4xlarge nodes --> 48 CPUs, 3 GPUs
 head_node_type:
   name: head_node_type
-  instance_type: g4dn.2xlarge
+  instance_type: g5.4xlarge
```

Comment:
why do we use g5? G5 uses a10 instead of t4?

Reply:
I see. It makes sense. T4 is mentioned in the template. We should replace it with a10.